content/ja/docs/setup/best-practices/certificates.md

```diff
@@ -110,7 +110,7 @@ You must manually configure these administrator account and service accounts:
 | admin.conf              | default-admin              | kubernetes-admin               | system:masters |
 | kubelet.conf            | default-auth               | system:node:`<nodeName>` (see note) | system:nodes   |
 | controller-manager.conf | default-controller-manager | system:kube-controller-manager |                |
-| scheduler.conf          | default-manager            | system:kube-scheduler          |                |
+| scheduler.conf          | default-scheduler          | system:kube-scheduler          |                |
```

Comment:
AFAIK, PRs should only target a single language - e.g. `/en`.

Reply:
@camper42 Please separate this PR by each language (en/ko/ja).